component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/x509Certificate/X509CertificateAuthenticator.java

```diff
@@ -522,6 +546,20 @@ private void validateUsingSubject(String subject, AuthenticationContext authenti
         }
     }
 
+    /**
+     * Check the authenticated user's tenant domain and verify whether it from super tenant or different tenant.
+     *
+     * @param authenticatedUser Get the authenticated user object from the authentication context  .
+     */
+    private String getAuthenticatedUserName(AuthenticatedUser authenticatedUser) {
+
+        String userName = authenticatedUser.getAuthenticatedSubjectIdentifier();
```

Comment:
Dont we need to a null check here?

Reply:
null check is handled from the place where this method is being called.